### DISCUSSION TITLE
Configure SearXNG as a retriever

### DISCUSSION THREAD

**Discussion by mdlmarkham** [assafelovic/gpt-researcher#862] [Q&A]
I am running SearXNG in my homelab and would like to configure it as a retriever in gptr. I see that searx is supported, but it's not being actively developed. SearXNG (https://github.com/searxng/searxng) is the active project. Has anyone done this? Any suggestions on how to configure the .env file? My instance is local, and doesn't have an API key.

**Comment by programmingninjas** [comment]
SEARX_URL="http://localhost:8080"
RETRIEVER="searx"

These are the env vars that you need to pass in order to get it work on searxng
